data/_models/shared/Resources.csv

```diff
@@ -73,7 +79,7 @@ Resources,,creator,ref_array,,,,Organisations,,,,dcterms:creator,This attribute
 Resources,,networks involved,refback,,,,Resources,,resources,,,The consortia or networks that this resource is part of,"DataCatalogueFlat,EMA"
 Resources,,networks other,text,,,,,,,,,List the names of any other networks that are not listed and this resource is involved in,EMA
 Resources,,available data and samples,heading,,,,,,,,,Data model and contents,"DataCatalogueFlat,EMA,RWEStaging,CohortsStaging"
-Resources,,datasets,refback,,,,Datasets,,resource,,,List of datasets contained in this resouce if applicable,"DataCatalogueFlat,CohortsStaging,RWEStaging,NetworksStaging,StudiesStaging"
+Resources,,datasets,refback,,,,Datasets,,resource,,"dcat:datasets",List of datasets contained in this resouce if applicable,"DataCatalogueFlat,CohortsStaging,RWEStaging,NetworksStaging,StudiesStaging"
```

Comment:
should this not be dcat:distribution, otherwise we will need all the mandatory fields of the dcat:Dataset in our Datasets table as well. I was thinking we should annotate our Datasets table with dcat:Distribution

Reply:
ah good question. But that would make things totally confusing. We probably should rename then 'Datasets' to become 'Data tables' (which it used to be called more or less before).

Reply:
Ok, tried it but then it need an accessURL to validate. For now removed the dcat:datasets annotation